modules/gapi/src/streaming/onevpl/utils.cpp

```diff
@@ -429,4 +432,156 @@ std::string ext_mem_frame_type_to_cstr(int type) {
 } // namespace wip
 } // namespace gapi
 } // namespace cv
+
+// FIXME: find a way to remove this
+#ifdef HAVE_DIRECTX
+#ifdef HAVE_D3D11
+// FIXME: consider handling it in cmake
+#pragma comment(lib,"d3d11.lib")
```

Comment:
You cannot link this library from cake. Pragma have to be here

Reply:
Removed